pkg/operator/ceph/pool/controller.go

```diff
@@ -390,7 +390,7 @@ func createPool(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo,
 	}
 	logger.Infof("initializing pool %q for RBD use", p.Name)
 	args := []string{"pool", "init", p.Name}
-	output, err := cephclient.NewRBDCommand(context, clusterInfo, args).Run()
+	output, err := cephclient.ExecuteRBDCommandWithTimeout(context, args)
```

Comment:
Instead, try this:
```suggestion
	output, err := cephclient.NewRBDCommand(context, clusterInfo, args).RunWithTimeout(exec.CephCommandsTimeout)
```

Reply:
it's ok,  after change the funcm that it is normal, why? @travisn

Reply:
`ExecuteRBDCommandWithTimeout()` has a different implementation than `NewRBDCommand(context, clusterInfo, args).Run()`, there must have been some different arguments that caused the failure.